packages-exp/functions-exp/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const path = require('path');
+
+module.exports = {
+  extends: '../../config/.eslintrc.js',
+  parserOptions: {
+    project: 'tsconfig.json',
+    // to make vscode-eslint work with monorepo
+    // https://github.com/typescript-eslint/typescript-eslint/issues/251#issuecomment-463943250
+    tsconfigRootDir: __dirname
+  },
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        'packageDir': [path.resolve(__dirname, '../../'), './'],
+        devDependencies: true,
+        peerDependencies: true
```

Comment:
Shouldn't peerDependencies to be allowed?

Reply:
This allows peerDependencies (will not show error on peerDependency import). Are you saying it should not be allowed?

Also this whole rule is a mess for monorepos, it looks like someone has submitted a PR to try to fix it but it has not been looked at yet: https://github.com/benmosher/eslint-plugin-import/issues/1650

Reply:
Right. I meant not allowed.

Reply:
Removed. I thought it would give me errors on imports from `@firebase/app-types-exp` but I guess it doesn't? I don't know how this rule works anymore.